.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   - "0.10"
 before_script:
   - cp config.js.example config.js
+  - npm run nsp
```

Comment:
I'd put it in the `after_script`, even after running the `coveralls` script, so that all tests and coverage reports are run even if there is a vulnerability that is not the PR author's fault and even if the coverage is bad and considered failing. And you still get your `nsp` report if test pass :-)

The only reason why `nsp` would not run is if the tests are failing, which is a valid cause to never merge and ask the author for changes anyway.